src/background_process.c

```diff
@@ -164,6 +164,16 @@ void *background_process(void *ptr)
 		    log_to_disk(true);
 		    break;
 
+		case QTCRENTRY:
+
+		    store_recv_qtc(lan_message+2);
+		    break;
+
+		case QTCSENTRY:
+
+		    store_sent_qtc(lan_message+2);
+		    break;
+		    
```

Comment:
Any time you call a function which is not previously defined (by function declaration or prototype) C guesses a funcction prototype by itself: int xyz (). Later on it may see that the function inn reality has a different prototypeand will complain.

But if the funcion is defined in a different *.c file the compiler has no chance to check for that and therefore has no chance to check it out.

In our case it was not critical because the compiler thoughed the function will return an int which we ignored. But the function did return void. But there may be subtle bugs - I am bitten more than once by it.